main.go

```diff
@@ -119,9 +118,6 @@ func main() {
 	//
 	viper.SetDefault("show_joinquit", false)
 	showJoinQuit := viper.GetBool("show_joinquit")
-	// Maximum length of user nicks aloud
-	viper.SetDefault("max_nick_length", ircnick.MAXLENGTH)
-	maxNickLength := viper.GetInt("max_nick_length")
```

Comment:
lets tell the user that it's not detected at runtime and they should remove this field from their config

Reply:
I'll add it like we did for `nickserv_identify` (or you can if you want, I won't be able to get back to this till tomorrow, life events).

Reply:
Done. In this one I merely inform the user, but don't actually exit the program. Let me know if we should tell the program to exit in this case.